src/tally.F90

```diff
@@ -120,6 +127,27 @@ subroutine score_general(p, t, start_index, filter_index, i_nuclide, &
         end if
 
 
+      case (SCORE_INVERSE_VELOCITY)
+        if (t % estimator == ESTIMATOR_ANALOG) then
+          ! All events score to a inverse velocity bin. We actually use a
+          ! collision estimator in place of an analog one since there is no way
+          ! to count 'events' exactly for the inverse velocity
+          if (survival_biasing) then
+            ! We need to account for the fact that some weight was already
+            ! absorbed
+            score = p % last_wgt + p % absorb_wgt
+          else
+            score = p % last_wgt
+          end if
+          score = score / material_xs % total &
+               / (sqrt(2 * p % E / (MASS_NEUTRON * AMU_MEV)) * C_LIGHT)
```

Comment:
Also, CODATA provides neutron mass energy equivalent (939.565379 MeV/c^2), so you may want to consider having a constant `MASS_NEUTRON_MEV` instead.